src/proof/bitvector_proof.h

```diff
@@ -9,161 +9,194 @@
  ** All rights reserved.  See the file COPYING in the top-level source
  ** directory for licensing information.\endverbatim
  **
- ** \brief Bitvector proof
+ ** \brief Bitvector proof base class
  **
- ** Bitvector proof
+ ** Contains code (e.g. proof printing code) which is common to all bitvector
```

Comment:
I thought that we wanted to move all printing code into a parallel class hierarchy. Am I remembering this wrong or is this going to be done later?